src/utils/DefaultValueContext.h

```diff
@@ -54,6 +54,11 @@ class DefaultValueContext final : public ExpressionContext {
         return Value::kEmpty;
     }
 
+    Value getColumn(int32_t) const override {
+        LOG(FATAL) << "Not allowed to call";
```

Comment:
`LOG(FATAL)` will take crash

Reply:
emm. it's ok when vesoft-inc/nebula-common#316. merged